From: Damien Goutte-Gattat via Gnupg-devel <gnupg-devel@gnupg.org>
Date: Wed, 11 Apr 2018 16:06:01 +0100
Subject: build: Make sure version.texi is generated in time.

[[PGP Signed Part:Good signature from AA4373E6C82C78AD Damien Goutte-Gattat <dgouttegattat@incenp.org> (trust undefined) created at 2018-04-11T17:05:27+0200 using RSA]]
[1. text/plain]

* doc/Makefile.am (yat2m-stamp): Depend on version.texi.
--

When building from a cloned Git repository and with `make -j 3`
(or higher), the version.texi file may not have been generated
yet when yat2m is called to generate the man page, resulting in
a build failure.

Signed-off-by: Damien Goutte-Gattat <dgouttegattat@incenp.org>
(cherry picked from commit 8ef76d9373db16f1ca176f9c87c8dac2758c8b85)
---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index d5eb886..7439a49 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -54,7 +54,7 @@ YAT2M_DEP = yat2m
 endif
 endif
 
-yat2m-stamp: $(myman_sources)
+yat2m-stamp: $(myman_sources) $(srcdir)/version.texi
 	@rm -f yat2m-stamp.tmp
 	@touch yat2m-stamp.tmp
 	for file in $(myman_sources) ; do \
